index.js

```diff
@@ -2017,6 +2017,24 @@
     return Foldable.methods.reduce(foldable)(f, x);
   }
 
+  //# foldMap :: (Monoid m, Foldable f) => (TypeRep m, a -> m, f a) -> m
+  //.
+  //. Deconstructs a foldable by mapping every element to a monoid and
+  //. concatenating the results.
+  //.
+  //. This function is derived from [`reduce`](#reduce), [`concat`](#concat)
+  //. and [`empty`](#empty).
+  //.
+  //. ```javascript
+  //. > foldMap(String, x => x.toString(), [1, 2, 3])
+  //. '123'
+  //. ```
+  function foldMap(typeRep, f, foldable) {
+    return reduce(function(b, a) {
+      return concat(b, f(a));
+    }, empty(typeRep), foldable);
```

Comment:
I'd prefer this formatting:

```javascript
return reduce(function(b, a) { return concat(b, f(a)); },
              empty(typeRep),
              foldable);
```

Also, I'd prefer to name the parameters `monoid` and `x` rather than `b` and `a`.